Source/WTF/wtf/Threading.cpp

```diff
@@ -57,6 +57,10 @@
 #endif
 #endif
 
+#if OS(WINDOWS)
+#include <wtf/RunLoop.h>
+#endif
```

Comment:
I don't think you need an `#if` here; the header isn't platform-specific, even if the usage happens to be.

Reply:
Fixed.